tests/pytests/data/V46475015EDR/V46475015EDR_isis3.lbl

```diff
@@ -0,0 +1,78 @@
+Object = IsisCube
+  Object = Core
+    StartByte   = 65537
+    Format      = Tile
+    TileSamples = 512
+    TileLines   = 456
+
+    Group = Dimensions
+      Samples = 1024
+      Lines   = 3648
```

Comment:
Reduce the number of lines in the test labels so that the output data is smaller.